### PR TITLE
Android fix

### PR DESCRIPTION
**Pull request by bakbanga**

### CODE REVIEW

File: www/js/app.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ angular.module('app', [
   $httpProvider.interceptors.push('authInterceptor');
   
   $ionicConfigProvider.views.transition('android');
+  //$ionicConfigProvider.scrolling.jsScrolling(false);
```
Comment (austinpapp) [inline www/js/app.js:22]: is this comment necessary ? future code?

File: www/js/controllers/petitions.js [hunk 2]
```diff
@@ -160,11 +160,11 @@ function ($scope,  petitions, PetitionsResource, groups, $stateParams, errorForm
       $scope.shareTitle = petition.title;
       $scope.shareImage = petition.share_picture;
 
-      if ((petition.answer_id && petition.answer_id !== 3) || petition.expired || session.user_id === petition.user.id) {
-        $scope.subview = 'templates/petitions/results.html';
-      } else {
+      //if ((petition.answer_id && petition.answer_id !== 3) || petition.expired || session.user_id === petition.user.id) {
```
Comment (austinpapp) [inline www/js/controllers/petitions.js:163]: can this be removed?

File: www/js/router.js [hunk 3]
```diff
@@ -1,275 +1,276 @@
...
+      url: '/logout',
+      templateUrl: 'templates/session/logout.html',
+      controller: 'session.logout'
+    }, {
+      name: 'commingSoon',
+      url: '/comming-soon',
+      cache: true,
+      templateUrl: 'templates/coming-soon.html'
+    }, {
+      name: 'poling',
```
Comment (austinpapp) [inline www/js/router.js:44]: polling?

Comment (bakbanga) [inline www/js/router.js:44]: It was 'poling' before(in non-ionic version).

File: www/js/controllers/petitions.js (hunk 2 above)
Comment (bakbanga) [inline www/js/controllers/petitions.js:163]: These were wrong-commented and have been reverted in next PR.

File: www/js/app.js (hunk 1 above)
Comment (bakbanga) [inline www/js/app.js:22]: Yes

File: www/js/router.js (hunk 3 above)
Comment (austinpapp) [inline www/js/router.js:44]: ok thanks. we may need to swing back to this one another time. if its a typo that is now functioning, we'll just keep with what had previously been there.

File: www/js/controllers/petitions.js (hunk 2 above)
Comment (austinpapp) [inline www/js/controllers/petitions.js:163]: ok thanks.

**Pull request merged**